server/tests/template.load

```diff
@@ -26,12 +26,17 @@ CAST column Channels.Type to "channel_type" drop typemod,
      type tinyint when (<= precision 4) to boolean using tinyint-to-boolean,
      type json to jsonb drop typemod
 
-MATERIALIZE VIEWS exclude_products
-     excluding table names matching ~<IR_>, ~<focalboard>
+EXCLUDING TABLE NAMES MATCHING ~<IR_>, ~<focalboard>
 
 BEFORE LOAD DO
-     $$ ALTER SCHEMA public RENAME TO {{ .source_schema }}; $$
+     $$ ALTER SCHEMA public RENAME TO {{ .source_schema }}; $$,
+     $$ DROP INDEX IF EXISTS idx_posts_message_txt; $$,
+     $$ DROP INDEX IF EXISTS idx_fileinfo_content_txt; $$
```

Comment:
What's the context behind these indexes?

Reply:
@streamer45 These are actually related to: https://docs.mattermost.com/deploy/postgres-migration.html#full-text-indexes matter where `pgloader` will error and abort. We are just running this as a query later on and hopefully get it done.

Reply:
Thanks :+1: